api/database/migrations/2019_10_23_175334_create_learning_materials_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateLearningMaterialsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('learning_materials', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('name');
+            $table->unsignedBigInteger('user_id');
+            $table->timestamps();
+
```

Comment:
[nitpick] Don't we need an url or something to reference the file where it is stored? It might not be obvious what it should be now since we don't know where we're storing them so this could be implemented in a later migration.

Reply:
Maybe it would be used as a name, but in that case column name 'name' is not valid. 
@ulorenas what you were planning?